README.md

```diff
@@ -70,10 +71,11 @@ For clarity, an example of the setup below can be found in the [ENS subgraph rep
     {
       "scripts": {
         "codegen": "graph codegen --output-dir types/",
+        "create-subgraph": "graph create <SUBGRAPH-NAME> --node http://127.0.0.1:8020",
         "build": "graph build",
-        "build-ipfs": "graph build --ipfs /ip4/127.0.0.1/tcp/5001",
+        "build-ipfs": "graph build --ipfs http://localhost:5001 subgraph.yaml",
         "deploy":
-          "graph deploy --ipfs /ip4/127.0.0.1/tcp/5001 --node http://127.0.0.1:8020 --subgraph-name <SUBGRAPH_NAME>"
+          "graph deploy --debug <SUBGRAPH-NAME> --ipfs http://localhost:5001/ --node http://127.0.0.1:8020"
```

Comment:
I'd put the `<SUBGRAPH-NAME>` at the end and make it `<SUBGRAPH_NAME>` (note the underscore).